docs/core/linux-prerequisites.md

```diff
@@ -32,6 +36,10 @@ This article shows the dependencies needed to develop .NET Core applications on
  * openSUSE 42.2+ x64
```

Comment:
PM team likes to use +, but we should avoid it because it's not an international concept. Replace with "or later versions". Also, do you need to call out the architecture? Aren't they all x64 anyway?